### DISCUSSION TITLE
Is scoop irrelevant?

### DISCUSSION THREAD

**Discussion by DemGiran** [Devolutions/UniGetUI#642] [General]
It doesn't make sense for Winget to coexist in a user interaction enhancement project on Winget in tandem with some other thing.

  Scoop is unique in what it does, but those two programs (Winget and Scoop) work in very different ways to achieve a similar goal. Aggregating package managers will ultimately hinder the long-term potential of what the WingetUI project could grow to be. With scoop out of the way, it could focus on being a one-stop-shop for the graphical management of software, an installer, an updater, an uninstaller, for everything on the computer, an enhanced version of the MSStore, literally.  And then there's Scoop, a powerful niche Chocolatey/Winget alternative. It doesn't do UAC, and it has some software Winget doesn't (and vice versa, but besides the point). It extracts the installers; it has its own portable-like folder for its apps. It's a different thing, should be irrelevant, imho.

**Comment by marticliment** [comment]
I don't think that's true. And there are people who actually like. wingetui because of this differentiation.

the next version will bring support with chocolatey, and a new window shown on the first run that will ask the user which package managers do they wsnt to use, so if you want only winget, or scoop, or chocolatey, you just need to disable them.

**Comment by marticliment** [comment]
I personally like the idea fo having an UI to handle them all. I can update/install/uninstall. everything from the same place in a few clicks. and the same goes with updating and installing new software.

If what you are looking for is a GUI for winget, Handy Winget is possibly the best choice for you, because it includes lots of winget-only features that wingetui does not support

**Reply by Spacellary** [reply]
I actually wish Chocolatey and Winget were more like scoop. Scoop is a nice source to have, and still a useful tool at our disposal. If supporting multiple sources ever become a burden then I'd understand the worries.
